column_operations/rasgo_datetrunc/rasgo_datetrunc.yaml

```diff
@@ -6,11 +6,10 @@ source_code: rasgo_datetrunc.sql
 description: Truncates a date to the datepart you specify. For example, if you truncate the date '10-31-2022' to the 'month', you would get '10-1-2022'.
 arguments:
   date_column:
-    type: column_list
+    arg_type: column_list
     description: names of column(s) you want to datetrunc
   date_part:
-    type: string
-    values: 'year','month','day','week','quarter','hour','minute','second','millisecond','microsecond'
```

Comment:
Why did we get rid of the values here? Don't we want to enable a drop-down of potential values?

Reply:
which i need to fix in the yaml :)